dashboard/config/marketing/announcements.json

```diff
@@ -1,6 +1,6 @@
 {
   "pages": {
-    "/home": "transformers-sept-2024",
+    "/home": "hoc-2024",
```

Comment:
just checking that we only want this for teachers, not students and teachers?

Reply:
I thought the same thing, but the language on this banner focuses on registering. I assume we'll switch the student banner for the November launch w/ the new activity.

Reply:
Makes sense!